crates/rpc/rpc-builder/src/metrics.rs

```diff
@@ -245,5 +300,5 @@ struct RpcServerCallMetrics {
     /// The number of failed calls
     failed_total: Counter,
     /// Response for a single call
-    time_seconds: Histogram,
```

Comment:
this might break existing existing metric setups, so I'd rather keep this unchanged